sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,258 @@
+      DefaultCPUAccounting=yes
+      DefaultMemoryAccounting=yes
+      CONF
+      sudo mkdir -p /etc/systemd/system.conf.d/
+      sudo mv origin-ami-accounting.conf /etc/systemd/system.conf.d/
+      sudo systemctl daemon-reexec
+  - type: "host_script"
+    title: "package the AMI"
+    script: |-
+      oct package ami --upgrade
+  - type: "script"
+    title: "use a ramdisk for etcd"
+    script: |-
+      sudo mkdir -p /tmp
+      sudo mount -t tmpfs -o size=2048m tmpfs /tmp
+      sudo restorecon -R /tmp
+  - type: "script"
+    title: "build an origin release"
+    repository: "origin"
+    script: |-
+      export OS_BUILD_IMAGE_ARGS=''
+      hack/build-base-images.sh
+      OS_ONLY_BUILD_PLATFORMS='linux/amd64' OS_BUILD_ENV_PRESERVE="_output/local/releases:_output/local/bin" hack/env make build-rpms
+      hack/build-images.sh
+      hack/extract-release.sh
+  - type: "script"
+    title: "verify commit history"
+    repository: "origin"
+    script: |-
+      # run commitchecker outside release container as it needs
+      # access to git; also explicitly force godeps verification
+      branch="$( git rev-parse --abbrev-ref --symbolic-full-name HEAD )"
+      if [[ "${branch}" == "master" ]]; then
+        RESTORE_AND_VERIFY_GODEPS=1 make verify-commits -j
+      fi
+  - type: "script"
+    title: "run check and verify tasks"
+    repository: "origin"
+    script: |-
+      OS_BUILD_ENV_EXCLUDE='_output/local/go' hack/env TEST_KUBE='true' JUNIT_REPORT='true' make check -o build -j -k
+  - type: "script"
+    title: "run integration tests"
+    repository: "origin"
+    script: |-
+      OS_BUILD_ENV_PRESERVE=_output/local/bin/linux/amd64/integration.test hack/env make build-integration-test
+      OPENSHIFT_SKIP_BUILD='true' JUNIT_REPORT='true' make test -o check -o build -k
+  - type: "host_script"
+    title: "prepare QE test queue"
+    script: |-
+      QE_TESTS_ARRAY=()
+      if [[ "$RUN_QE_FULLFUNCTIONAL_TESTS" == "true" ]]; then
+        QE_TESTS_ARRAY+=(fullfunction)
+      elif [[ "$RUN_QE_ACCEPTANCE_TESTS" == "true" ]]; then
+        QE_TESTS_ARRAY+=(acceptance)
+      elif [[ "$RUN_QE_SMOKE_TESTS" == "true" ]]; then
+        QE_TESTS_ARRAY+=(smoke)
+      fi
+
+      if [[ ${#QE_TESTS_ARRAY[@]} -eq 0 ]]; then
+        exit 0
+      fi
+
+      QE_TESTS=""
+      for test in "${QE_TESTS_ARRAY[@]}"; do
+        if [[ -z "$QE_TESTS" ]]; then
+          QE_TESTS=$test
+        else
+          QE_TESTS=$QE_TESTS,$test
+        fi
+      done
+
+      QE_TEST_QUEUE=~/fork_ami_qe_test_queue
+      if [[ -n "$QE_TESTS" && -f "$QE_TEST_QUEUE" ]]; then
+        AMI_ID=$(ssh -F ./.config/origin-ci-tool/inventory/.ssh_config openshiftdevel curl -s http://169.254.169.254/latest/meta-data/ami-id)
+        sed -i "/${AMI_ID}.*/d" $QE_TEST_QUEUE
+        echo "${AMI_ID}=$QE_TESTS" >> $QE_TEST_QUEUE
+      fi
+  - type: "host_script"
+    title: "release the AMI"
+    script: |-
+      oct package ami --mark-ready --tag qe=ready
```

Comment:
You can already set a Trello URL in Jenkins, I will also update this job to set an additional tag in the AMI. Bugzilla ID SGTM but I would prefer to do that in a follow-up since it's an extra feature not supported by the old fork_ami job.